src/mono/wasm/debugger/BrowserDebugProxy/MonoProxy.cs

```diff
@@ -229,6 +243,22 @@ protected override async Task<bool> AcceptEvent(SessionId sessionId, JObject par
 
             return false;
         }
+
+        protected void CreateWorkerExecutionContext(SessionId workerSessionId, SessionId originSessionId)
+        {
+            if (!contexts.TryGetValue(originSessionId, out ExecutionContext context))
+            {
+                logger.LogDebug($"Origin sessionId does not exist - {originSessionId}");
+                return;
+            }
+            if (contexts.ContainsKey(workerSessionId))
+            {
+                logger.LogDebug($"Worker sessionId already exists - {originSessionId}");
```

Comment:
This shouldn't happen, right? Since, multithreaded debugging is WIP right now, it might be more useful to throw an exception here.
Also, we should handle `Runtime.executionContextCreated`, and `Runtime.executionContextsCleared` to remove them from `contexts`. And `contexts` might need to handle being modified in parallel. This can be done in a follow up PR though.